frontend/src/theme/base.scss

```diff
@@ -25,3 +25,7 @@ body {
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
+
```

Comment:
Since you are applying a style to a component (table), please put this style inside components.scss file, and without the div as prefix